node_monitor/bot_telegram.py

```diff
@@ -9,11 +9,12 @@ def __init__(self, telegram_token: str) -> None:
 
 
     def send_message(
-            self, chat_id: str, message: str
+            self, chat_id: str, subject: str, message: str
         ) -> None | requests.exceptions.HTTPError:
         """Send a message to a single Telegram chat."""
+        dispatch = f"{subject}\n\n{message}"
         max_message_length = 4096
-        message_parts = textwrap.wrap(message, width=max_message_length)
+        message_parts = [dispatch[i:i + max_message_length] for i in range(0, len(dispatch), max_message_length)]
```

Comment:
I would recommend defining and using `batched` here, but i'll let you decide what you want to do. If you decide to keep it this way, please add a comment `TODO: use itertools.batched here when python version is updated to >=3.12`.

If you decide to define `batched` here, please add a comment `TODO: delete this for itertools.batched when python version is updated to >=3.12`.